src/dotnet/CarbonAware.LocationSources.Azure/src/AzureLocationSource.cs

```diff
@@ -28,45 +29,7 @@ public class AzureLocationSource : ILocationSource
     public AzureLocationSource(ILogger<AzureLocationSource> logger)
```

Comment:
style-👮🏽: The order of functions in this file isn't quite dotnet standards. It makes more sense to have public functions first with private/internal ones later. You can see this SO doc for a good guide on what ordering is generally used: https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods